src/queries/term-level-queries/wildcard-query.js

```diff
@@ -25,6 +25,22 @@ class WildcardQuery extends MultiTermQueryBase {
         super('wildcard', field, value);
     }
 
+    /**
+     * Allow case insensitive matching or not (added in 7.10.0).
```

Comment:
Could you please add this method to the type declaration here - https://github.com/sudo-suhas/elastic-builder/blob/v2.13.0/src/index.d.ts#L1755

Reply:
yes, updated!